apps/src/code-studio/progressRedux.js

```diff
@@ -556,12 +556,9 @@ const peerReviewLevels = state =>
   state.peerReviewLessonInfo.levels.map((level, index) => ({
     // These aren't true levels (i.e. we won't have an entry in levelResults),
     // so always use a specific id that won't collide with real levels
+    ...level,
```

Comment:
Just checking -- is level.icon only set when level.locked is true?

Reply:
yeah.